docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Sanitize databases on non-production environments
+sidebarTitle: Sanitize databases
+description: Find out how to sanitize databases on non-production environments.
+---
+
+<!-- 
+When to use
+  When there is a single outcome a user wants to achieve.
+  When you want to explain how to get to the outcome in ordered steps.
+  https://diataxis.fr/how-to-guides/ 
+
+How to use
+  1. Copy this template into the right directory in /src/docs/.
+  2. Rename it to match the title.
+  3. Replace the following content with your own.
+-->
+
+When creating or pushing a branch to Platform.sh, both the code and the database get inherited to a new branch.
+Depending on your processes, internal or external teams may check the code changes on that new branch.
+
+Databases of live websites often contain Personally Identifiable Information (PII) such as full name, mailing addresses, ... .
+To avoid any unauthorized access to PII on the development environment, you can sanitize the database.
+
+## Before you begin
+
+This guide is about sanitizing databases.
+This guide doesn't address neither input validation nor input sanitization, an introduction to these topics can be found in the (free) [Open Web Application Security Project (OWASP) cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/Input_Validation_Cheat_Sheet.html#goals-of-input-validation).
+The `log4j` security vulnerability is an example of what missing input sanitization can cause.
+
+NoSQL Databases ([MongoDB](../configuration/services/mongodb/_index.md), ...) can benefit from sanitization but for the sake of simplicity, 
+these use cases won't be covered in this guide.
+
+You'll need:
+
+- a project that uses a [MySQL database](../configuration/services/mysql/_index.md),
+- the [Platform CLI](/development/cli/_index.md#cli-command-line-interface) installed locally.
+
+## Sanitize the Database
+
+Make sure that the sanitization only runs on the development environments and **never** on the production environment.
+Otherwise you will loose most — if not all — of the relevant data stored in your database.
```

Comment:
```suggestion
Otherwise you may lose most or even all of the relevant data stored in your database.
```
"if not all" is a nice phrase but not the most clear one for an international audience. Especially things with negatives can be hard.